packages/client/src/pages/ForumTopic/ForumTopic.tsx

```diff
@@ -2,37 +2,66 @@ import { useSidebarMode } from '@utils'
 import Comment from './elements/Comment'
 import InputLine from './elements/InputLine'
 import styles from './forumTopic.module.css'
-import { FORUM_TOPIC_COMMENT_STUB, FORUM_TOPIC_STUB } from './stub'
+import { useAppDispatch, useAppSelector } from '@store/hooks'
+import { forumSelectors } from '@store/forum'
+import Loader from '@components/Loader'
+import Empty from '@components/Empty'
+import { useLocation } from 'react-router'
+import { useEffect } from 'react'
+import { getTopic } from '@store/forum/thunk'
 
 const ForumTopic = () => {
-    const { title, description, author, date } = FORUM_TOPIC_STUB
+    const topicData = useAppSelector(forumSelectors.selectTopicData)
+
+    const loadStatus = useAppSelector(forumSelectors.selectStatus)
+    const isLoading = loadStatus === 'loading'
+
+    const dispatch = useAppDispatch()
+    const location = useLocation()
```

Comment:
В react-router есть готовая функция `useParams`
```suggestion
    const { id: topicId } = useParams()
```